src/main/fc/fc_init.c

```diff
@@ -478,8 +478,9 @@ void init(void)
     initBoardAlignment(boardAlignment());
 
     if (!sensorsAutodetect()) {
-        // if gyro was not detected due to whatever reason, we give up now.
-        failureMode(FAILURE_MISSING_ACC);
+        // if gyro was not detected due to whatever reason, notify and don't arm.
+        failureLedCode(FAILURE_MISSING_ACC, 2);
```

Comment:
I am not sure of the value of beeping at this stage. I am adding code to indicate the reason for not arming when arming is attempted, and 'no gyro' is one of these reasons. The way the code works now, 'no gyro' has simply stopped being a reason for boot failure.